jormungandr/src/utils/task.rs

```diff
@@ -118,6 +119,49 @@ impl Services {
         self.services.push(task);
     }
 
+    /// Spawn the given Future in a new dedicated runtime
+    pub fn spawn_try_future_std<F, T>(&mut self, name: &'static str, f: F)
```

Comment:
Why do that if we already have `spawn_failable_std`?

Reply:
because that's for a task, not for a service.